values/monitoring/grafana-goquorum-dashboard.yml

```diff
@@ -0,0 +1,2220 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-goquorum-dashboard
+  namespace: default
+  labels:
+    grafana_dashboard: "1"
+data:
+    grafana-goquorum-dashboard.json: |-
+      {
+        "annotations": {
+          "list": [
+            {
+              "builtIn": 1,
+              "datasource": "-- Grafana --",
+              "enable": true,
+              "hide": true,
+              "iconColor": "rgba(0, 211, 255, 1)",
+              "name": "Annotations & Alerts",
+              "target": {
```

Comment:
Do we need this? Are we deploying geth?

Reply:
https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/55610c410186a352fe848b61fb5761986a4f0309